src/runtime/opencl.cpp

```diff
@@ -687,16 +687,19 @@ WEAK int halide_opencl_initialize_kernels(void *user_context, void **state_ptr,
         } else {
             debug(user_context) << (void *)program << "\n";
         }
-        (*state)->program = program;
 
+        // halide_print(user_context, "Source: \n");
```

Comment:
Please delete commented-out code, or add a comment explaining why it's commented out